examples/13-streamlines/00_plot_3d_streamlines.py

```diff
@@ -79,6 +79,14 @@
 )
 pl1.show_figure(show_axes=True)
 
+###############################################################################
```

Comment:
Look the following commit to see how I tried to get the animation working for streamlines: https://github.com/ansys/pydpf-core/pull/1077/commits/90f9ef33ac5317aa5020b6afbc14873a8813b6cc. It's not working because it look either there's a bug in the "add_field" method or in the streamlines Field object.